.github/workflows/firebase-hosting-merge.yml

```diff
@@ -1,11 +1,8 @@
-# This file was auto-generated by the Firebase CLI
+# This file was initially auto-generated by the Firebase CLI
 # https://github.com/firebase/firebase-tools
 
 name: Deploy to Firebase Hosting on merge
-'on':
-  push:
-    branches:
-      - master
+on: push
```

Comment:
Run this workflow on every PR cause some noise. e.g. https://github.com/GoogleContainerTools/kpt-functions-catalog/pull/288 The CI is actually fine except this workflow is failing. 
Ideally, we should filter by branches like:
```
  push:
    branches:
      - master
      - [-\w]*\/v\d*\.\d*  # I assume GHA supports filtering branches by regex.
```

Reply:
@etefera

Reply:
Addressed in https://github.com/GoogleContainerTools/kpt-functions-catalog/pull/292

Reply:
Thanks for the quick response!